tools/build_travis.py

```diff
@@ -137,6 +137,30 @@
                "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
                "usb"  : ["USB_1", "USB_2" ,"USB_3"],
                }
+     },
+    {"target": "K64F",
+     "toolchains": "GCC_ARM",
+     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
+               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
```

Comment:
Looks like Travis isn't happy about the `"fat"` tests either. It's saying that the pins aren't defined.

The following tests need the pins set just for the KL43Z:
- [MBED_A12](https://github.com/ARMmbed/mbed-os/blob/646cc89a620f978ed6174cb7bbb921e0f0d910f2/features/unsupported/tests/mbed/sd/main.cpp)
- [PERF_1](https://github.com/ARMmbed/mbed-os/blob/646cc89a620f978ed6174cb7bbb921e0f0d910f2/features/unsupported/tests/mbed/sd_perf_stdio/main.cpp)
- [PERF_2](https://github.com/ARMmbed/mbed-os/blob/646cc89a620f978ed6174cb7bbb921e0f0d910f2/features/unsupported/tests/mbed/sd_perf_fhandle/main.cpp)
- [PERF_3](https://github.com/ARMmbed/mbed-os/blob/646cc89a620f978ed6174cb7bbb921e0f0d910f2/features/unsupported/tests/mbed/sd_perf_fatfs/main.cpp)

And the following tests need the pins set for the K64F, K22F, KL43Z.
- [MBED_19](https://github.com/ARMmbed/mbed-os/blob/646cc89a620f978ed6174cb7bbb921e0f0d910f2/features/unsupported/tests/mbed/dir_sd/main.cpp)

Alternatively, you can disable the `"fat"` checks for these platforms.